unit_test/conftest.py

```diff
@@ -27,4 +28,11 @@ def mock_path_exists(path):
     args = ["cibuildwheel", str(MOCK_PACKAGE_DIR)]
     monkeypatch.setattr(Path, "exists", mock_path_exists)
     monkeypatch.setattr(sys, "argv", args)
-    return args
+
+    tmp_path.joinpath(MOCK_PACKAGE_DIR).mkdir()
+    cwd_backup = os.getcwd()
+    os.chdir(tmp_path)
```

Comment:
`monkeypatch.chdir(path)` should work here, which would save a little cleanup code.

I'm curious - is this related to the 3.7 changes or incidental?

Reply:
I applied your suggestion & made a different commit for the explanation on this one.